mesh/ncmesh.cpp

```diff
@@ -599,6 +600,18 @@ void Refinement::SetType(char type, real_t scale)
    SetScale(ScaledType(type, scale));
 }
 
+int Refinement::GetNumChildren(Geometry::Type geom_type, char ref_type)
+{
+   if (geom_type != Geometry::PYRAMID)
+   {
+      return ref_type_num_children[int(ref_type)];
```

Comment:
Isn't this only for `geom_type == CUBE`? Should it return `8` for `TETRAHEDRON`, etc.? Also, if `ref_type_num_children` is only for hexahedra, should we rename it with the prefix `hex_`?